### DISCUSSION TITLE
USB sharing option not available

### DISCUSSION THREAD

**Discussion by sbng-dn** [utmapp/UTM#5368] [UTM Help (macOS)]
UTM version 4.2.5
MacOS ventura 13.4
Guest OS: windows 11

Everything is working fine but when I was trying to navigate and edit the config to support external USB devices to the windows11 VM, the USB sharing (as documented in the documentation was not available). How can I get USB sharing working as documented? 
![utm](https://github.com/utmapp/UTM/assets/125443552/53a8d042-c7ed-4bff-a059-bb5dade07c09)

**Comment by osy** [comment]
https://docs.getutm.app/settings-qemu/input/
